tests/network/vmi_multus.go

```diff
@@ -789,6 +789,11 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		checkKubevirtResourceNameEnvVariable := func(vmi *v1.VirtualMachineInstance, networkName, sriovResourceName string) error {
```

Comment:
I think `validate...` is describe better what this function does, sometimes `check..` is too generic IMO.

Following the thread https://github.com/kubevirt/kubevirt/pull/6113#discussion_r678317112
I would extract this to regular function because there is no benefit in defining it as a lambda.
As far as I understand there is no variable the being set only on this context, 
all the variables are being passed to it as arguments.
Also it will reduce the noise on this test suite a bit.

Other labmda's on this context are also questionable if they should be lambda func's IMO...

Reply:
I got rid of this lambda function completely just by getting the vmi pod in `validatePodKubevirtResourceName`, which makes more sense.